packages/flutter/lib/src/cupertino/context_menu.dart

```diff
@@ -204,13 +349,15 @@ class _CupertinoContextMenuState extends State<CupertinoContextMenu> with Ticker
   Rect? _decoyChildEndRect;
   OverlayEntry? _lastOverlayEntry;
   _ContextMenuRoute<void>? _route;
+  final double _midpoint = CupertinoContextMenu.animationOpensAt/2;
```

Comment:
Nit: Space out the math (just going based on what I usually see in the framework): `At / 2`